protos/index.proto

```diff
@@ -108,6 +108,9 @@ message DiskAnn {
 
   // L parameter
   uint32 L = 5;
+
+  /// Entry points to the graph
```

Comment:
these are the vertex id's?

Reply:
These are the IDs in the graph. Basically the "index" in the graph file. Not the "row id" in original dataset